CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+# Contribution guidance
+This page documents the procedures for:
+1. [Requesting new notebooks](#requesting-new-notebooks) to be added to the [IOOS CodeLab](https://ioos.github.io/ioos_code_lab);
+1. [Building](#building-new-notebooks) new notebooks;
+1. [Reviewing](#reviewing) submitted notebooks;
+1. [Merging](#merging) approved notebooks for inclusion into the Code Gallery;
+1. [Announcing](#announcing) the new notebooks to the community; and 
+1. [Logging any bugs](#logging-bugs) that are found in the Code Gallery.
+1. Managing the IOOS Data Demo Center [Project Board](#managing-ioos-data-demo-center-project-board).
+1. [Local management](#local-management-of-notebooks_demo) of the notebooks_demo repository.
+
+## Requesting new notebooks
+* New notebooks should be requested by adding a [ticket](https://github.com/ioos/ioos_code_lab/issues/new) to this GitHub repository. The ticket should contain the following information:
+```
+- [ ] What is language(s) for used in the example?
+- [ ] Is it focused on a particular module/software or an IOOS data service?
+- [ ] Can you provide a minimum example of the expected code and results in a notebook?
+
+Please provide a detailed description of the suggested example below:
+```
+
+### For the admin
+When the ticket gets created, make the following additions to the ticket:
+* Add the label [Notebook idea](https://github.com/ioos/ioos_code_lab/labels/Notebook%20idea).
+* Add the ticket to the [IOOS CodeLab](https://github.com/orgs/ioos/projects/1#card-49928448) project.
+* Put the card in the column [backlog](https://github.com/orgs/ioos/projects/1#column-5010196).
+
+## Building new notebooks
+If you would like to work on a notebook that has been proposed, follow these steps:
+1. [Create a fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of the [notebooks_demos](https://github.com/ioos/ioos_code_lab) repository.
+1. On your fork, [create a branch](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository) where you will start working on the new notebook. 
+   1. Give the branch a name that conveys what the new notebook is. For example, a notebook on converting data into Darwin Core could have a branch name of `data2dwc`. This makes it easier to know which branch is dedicated to which new notebook.
+1. On the new branch in your fork, start creating the new notebook. 
+   1. Notebooks are kept in the [jupyterbook/content/Code Gallery](https://github.com/ioos/ioos_code_lab/tree/master/jupyterbook/content/Code%20Gallery) directory of this repository.
```

Comment:
```suggestion
   1. Notebooks are kept in the [jupyterbook/content/code_gallery](https://github.com/ioos/ioos_code_lab/tree/master/jupyterbook/content/code_gallery) directory of this repository.
```

Reply:
This will be changed in #10 and will make our lives a bit easier. That space was causing a lot of trouble!